WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -149,8 +149,8 @@ private extension StoreStatsAndTopPerformersViewController {
             self.syncSiteVisitStats(for: siteID,
                                     siteTimezone: timezoneForSync,
                                     timeRange: vc.timeRange,
-                                    latestDateToInclude: latestDateToInclude) { error in
-                if let error = error {
+                                    latestDateToInclude: latestDateToInclude) { result in
+                if case let .failure(error) = result {
```

Comment:
nit: same here, I suggest using the `switch`

Reply:
Do you mean something like this? That's 4 lines instead of 1 🙂
```suggestion
                switch result {
                case .success:
                    break
                case .failure(let error):
```